.github/workflows/app.yml

```diff
@@ -68,7 +68,7 @@ jobs:
             }
   app-build:
     name: Build
-    runs-on: macos-26
+    runs-on: namespace-profile-default-macos
```

Comment:
we don't run this that often and it's quite fast – and to be able to iterate quickly on the QA prompt, it's useful not to wait ages for a GitHub runner